XamlControlsGallery/ControlPages/AnimatedVisualPlayerPage.xaml

```diff
@@ -0,0 +1,107 @@
+﻿<Page
+    x:Class="AppUIBasics.ControlPages.AnimatedVisualPlayerPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:animatedvisuals="using:AnimatedVisuals"
+    xmlns:local="using:AppUIBasics"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:muxcontrols="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <Grid>
+        <local:ControlExample x:Name="AnimatedVisualPlayerExample" 
+                              HeaderText="Playback of a Lottie animation."
+                              >
+
+            <local:ControlExample.Example>
+                <StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
+                    <!-- Text -->
+                    <TextBlock Width="600" TextWrapping="WrapWholeWords">
+                        This AnimatedVisualPlayer consumes a Lottie animation which was created in 
+                        Adobe AfterEffects and translated into Windows.UI.Composition objects using 
+                        <Hyperlink NavigateUri="https://aka.ms/lottie">
+                           Lottie-Windows
+                        </Hyperlink>.
+                        Since the 
+                        <Hyperlink NavigateUri="https://docs.microsoft.com/uwp/api/windows.ui.composition.compositionshape">
+                            CompositionShapes
+                        </Hyperlink> being used in the animation are supported on Windows 10 version 17763 and above, 
+                        the AnimatedVisualPlayer instatiates an Image as FallbackContent when its Source is unavailable.
+                    </TextBlock>
+                    <Border Width="400" Height="400" 
+                            BorderBrush="{ThemeResource ToolTipBorderThemeBrush}"
+                            BorderThickness="5"
+                            Background="LightGray"
+                            Margin="0, 20, 0, 20"
+                            >
+
+                        <!--  AnimatedVisualPlayer  -->
+                        <muxcontrols:AnimatedVisualPlayer x:Name="Player"
```

Comment:
Please include this chunk of markup as the example xaml markup.

Reply:
Unclear what's needed here -- @stmoy can you clarify or cite the relevant pattern I need to follow?

Reply:
seems like you can't have it inside a stackpanel. Instead, I've added the local:ControlExample.Xaml element to make things clear